src/cljs/rems/actions/request_comment.cljs

```diff
@@ -2,42 +2,29 @@
   (:require [re-frame.core :as rf]
             [reagent.core :as r]
             [rems.actions.action :refer [action-button action-form-view action-comment button-wrapper]]
-            [rems.atoms :refer [textarea]]
+            [rems.atoms :refer [enrich-user textarea]]
             [rems.autocomplete :as autocomplete]
-            [rems.status-modal :refer [status-modal]]
+            [rems.status-modal :as status-modal]
             [rems.text :refer [text]]
             [rems.util :refer [fetch post!]]))
 
-(defn fetch-potential-commenters
-  [[user on-success]]
+(rf/reg-fx
+ ::fetch-potential-commenters
+ (fn [[user on-success]]
   (fetch (str "/api/applications/commenters")
          {:handler on-success
```

Comment:
does on-success really need to be injected? why not hardcode `::set-potential-commenters`?

Reply:
Probably not. I left a couple of these unchanged. Something for later PR.